AnnoyancesFilter/sections/widgets.txt

```diff
@@ -592,7 +592,8 @@ rasmussen.edu##.embeddedServiceHelpButton
 ||sklum.com/core/controllers/front/LiveChat/js/OctoneController.js
 sklum.com##.oct8ne-btn
 schlaganfall-hilfe.de##.chatbot
-||service.force.com/embeddedservice/*/esw.min.js$domain=flyeralarm.com|hospitaloswaldocruz.org.br
+||service.force.com/embeddedservice/*/esw.min.js$domain=flyeralarm.com|hospitaloswaldocruz.org.br|ultrafarma.com.br
+||ultrafarma.my.salesforce.com/embeddedservice/*/esw.min.js^$domain=ultrafarma.com.br
```

Comment:
You can also remove the rule for `rasmussen.edu`.
I think your rules in that way should be ok, please check them.
```
||service.force.com/embeddedservice/*/esw.min.js
||my.salesforce.com/embeddedservice/*/esw.min.js
```